src/coreclr/vm/jitinterface.cpp

```diff
@@ -7868,6 +7868,38 @@ bool CEEInfo::haveSameMethodDefinition(
     return result;
 }
 
+CORINFO_CLASS_HANDLE CEEInfo::getTypeDefinition(CORINFO_CLASS_HANDLE type)
+{
+    CONTRACTL {
+        NOTHROW;
+        GC_TRIGGERS;
+        MODE_PREEMPTIVE;
+    } CONTRACTL_END;
+
+    CORINFO_CLASS_HANDLE result = NULL;
+
+    JIT_TO_EE_TRANSITION_LEAF();
```

Comment:
This should be full `JIT_TO_EE_TRANSITION`

Reply:
Fixed in f9a9aa24cbf7b90cbe4c6427dbd1d583eea145fb.